ptbcontrib/roles/roleshandler.py

```diff
@@ -28,18 +29,65 @@
 """:obj:`str`: The key used to store roles in ``bot_data``."""
 
 
+class RolesBotData(ABC):
+    """
+    Fill me
+
+    Defines the abstract class RolesBotData.
+    It has two abstract methods: get_roles and set_roles
```

Comment:
Abstract base class providing an interface to be used :meth:`set_roles`.
Inherit from this class when using a custom type for ``bot_data``